Architecture/Cloud-Services.md

```diff
@@ -73,11 +74,11 @@ Use Case
 Summary
 -------
 
-The web user works through a user agent to access resources at an RP.
+The web user works through a user agent to access resources at a RP.
```

Comment:
Because English is weird, whether to use "a" or "an" depends on whether the acronym starts with a vowel sound or not. So, in this case, "an RP" is still correct.